stories/landsat7.eocloud.stories.js

```diff
@@ -1,5 +1,7 @@
 import moment from 'moment';
```

Comment:
moment leftover

Reply:
Thanks, removed

Reply:
Thanks, removed.

Reply:
I already merged as the changes were needed in the other branch